providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/Region.java

```diff
@@ -54,13 +54,17 @@
    BRAZIL_SOUTH("Brazil South", "BR"),
    AUSTRALIA_EAST("Australia East", "AU-NSW"),
    AUSTRALIA_SOUTH_EAST("Australia Southeast", "AU-VIC"),
+   AUSTRALIA_CENTRAL("Australia Central", "AU-ACT"), 
+   AUSTRALIA_CENTRAL_2("Australia Central 2", "AU-ACT2"),
```

Comment:
The ISO code a standard code and is the same for those regions if they are geographically in the same place. Change to `AU-ACT`.

Reply:
Done! Sorry!